src/App.js

```diff
@@ -250,9 +250,17 @@ class App extends Component {
     render = () => {
         const {schema, uiSchema, formData, hasError, hasSuccess, submissionStatus} = this.state;
         const onSubmit = ({formData}) => this.submitForm(formData);
+	
+        // this.props.children come through as a DocumentFragment, 
+        // and cannot be added directly as a React element.
+        let tempElem = document.createElement("div");
+        tempElem.appendChild(this.props.children);
+	let childrenHtmlStr = tempElem.innerHTML;
+	console.log("this.props.children is: ["+ childrenHtmlStr + "]");
```

Comment:
What if we use this tool instead?  https://www.npmjs.com/package/react-html-parser

Reply:
Looks to nicely remove the need for `dangerouslySetInnerHTML`.  The optional `transform` and `convert` method hooks didn't seem to have a clean way to include converting the `DocumentFragment`.